Java/AutoML/automl-component/src/main/java/com/automl/component/model/Dataset.java

```diff
@@ -0,0 +1,99 @@
+package com.automl.component.model;
+
+import org.apache.commons.lang3.builder.ToStringBuilder;
+import org.apache.commons.lang3.builder.ToStringStyle;
+
+/**
+ * Dataset
+ * @author wangqingwei
+ * Created on 2022-08-09
+ */
+public class Dataset {
+
+    private long id;
+    private String name;
+    private String description;
+    private long namespaceId;
+    private String uri;
+    private long createId;
```

Comment:
Do you mean creatorId here?

Reply:
The person who created the dataset.
or you think this 'createId' name error?

Reply:
Yes, then it should be creatorId.

Reply:
OK，I've learned， I will modify Table Design and class